### ISSUE TITLE
HTML-AAM: aria mapping for @hidden is blank

### ISSUE THREAD

**Issue by jasonkiss** [w3c/html-aam#38]
_From @cyns on March 31, 2016 19:46_

Should it be no corresponding aria property?

_Copied from original issue: w3c/aria#315_

**Comment by jasonkiss** [comment]
Implicit semantics for `hidden` is `aria-hidden=true`, so that should be the ARIA mapping, and all APIs use the WAI-ARIA mapping.

**Comment by jasonkiss** [comment]
Updated mappings in 0e0f872 to reuse `aria-hidden=true` mappings. Closing.
